packages/server/src/api/routes/public/index.ts

```diff
@@ -8,18 +8,46 @@ import authorized from "../../../middleware/authorized"
 import { paramResource, paramSubResource } from "../../../middleware/resourceId"
 import { CtxFn } from "./utils/Endpoint"
 import mapperMiddleware from "./middleware/mapper"
+import env from "../../../environment"
+// below imports don't have declaration files
 const Router = require("@koa/router")
+const RateLimit = require("koa2-ratelimit").RateLimit
+const Stores = require("koa2-ratelimit").Stores
 const {
   PermissionLevels,
   PermissionTypes,
 } = require("@budibase/backend-core/permissions")
+const { getRedisOptions } = require("@budibase/backend-core/redis").utils
 
 const PREFIX = "/api/public/v1"
+const DEFAULT_API_LIMITING = 120
```

Comment:
Can we make this more explicit to what it represents? i.e. `DEFAULT_API_LIMITING_REQUESTS_PER_MINUTE`
It might make sense to model this as requests per second instead at that is a more useful time key when it comes to api load. Can update the internal unit below to `sec` to do this